crates/ruff_linter/src/checkers/ast/mod.rs

```diff
@@ -1853,6 +1841,10 @@ impl<'a> Checker<'a> {
             self.semantic.scope_id
         };
 
+        if self.semantic.in_exception_handler() {
+            flags |= BindingFlags::IN_EXCEPT_HANDLER;
+        }
```

Comment:
Idly wondering if we should just store the semantic model flags on the binding, in theory it's tiny.

Reply:
As in, get rid of the separate `BindingFlags` struct altogether, and store the semantic-model flags on `Binding` instead? Or store the semantic-model flags in addition to the `BindingFlags`?

Reply:
Store them in addition (and then remove this flag from `BindingFlags`). But, it's ok, no need to change.